doc/api/deprecations.md

```diff
@@ -634,6 +634,36 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP00XX"></a>
+### DEP00XX: crypto.createCipher()
+
+Type: Runtime
+
+[`crypto.createCipher()`][] generates keys from strings in an insecure manner,
+and, when used with a cipher that utilizes an initialization vector, will
+dangerously re-use initialization vectors. As such, it is immediately marked as
+deprecated, and will be fully removed in a later version.
+
+[`crypto.createCipheriv()`][] should be used in place of
+[`crypto.createCipher()`][]. Since [`crypto.createCipheriv()`][] will no longer
+attempt to derive a proper encryption key from a string, you must use a
+key-derivation function such as [`crypto.pbkdf2()`][] to obtain a valid key if
+you normally supply a string to [`crypto.createCipher()`][].
+
+If the previous key-derivation is required for backward compatiability, the new
+APIs [`crypto.generateLegacyKey()`][] and [`crypto.generateLegacyIV()`][] have
+been added.
+
+Additionally, for ciphers that require an initialization vector, a proper-length
+initialization vector must be passed to [`crypto.createCipheriv()`][].
+Initialization vectors must never be re-used, especially in modes such as
```

Comment:
`must` → `should`. The word `must` indicates that calling the function with such parameters leads to undefined behavior or an error, while this is actually just a strong recommendation.